benchmarks/perf-tool/release-configs/faiss-hnsw/test.yml

```diff
@@ -0,0 +1,32 @@
+endpoint: localhost
+test_name: "Faiss HNSW Test"
+test_id: "Faiss HNSW Test"
+num_runs: 10
+show_runs: false
+steps:
+  - name: delete_index
+    index_name: target_index
+  - name: create_index
+    index_name: target_index
+    index_spec: /home/ec2-user/[PATH]/index.json
```

Comment:
Shall we use `/home/ec2-user` by a root path parameter instead of hardcoded?

Reply:
It will require changes in the core perf tool. We can take this as a backlog. It was already like this on the test.yml so I am using it as it is.